From e14f4a1dd2b79f9bb7cebe028771363f45a39fc3 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Mon, 14 Aug 2017 17:59:05 +0000
Subject: [PATCH] MANATEE-351 Update to pg_repack v1.4.1 Reviewed by: Sean
 Chittenden <sean.chittenden@joyent.com> Reviewed by: David Pacheco
 <dap@joyent.com> Approved by: David Pacheco <dap@joyent.com>

---
 deps/pg_repack | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/pg_repack b/deps/pg_repack
index cac8179..20f5bf4 160000
--- a/deps/pg_repack
+++ b/deps/pg_repack
@@ -1 +1 @@
-Subproject commit cac817929901796e1ed18f9592694a355464d9b3
+Subproject commit 20f5bf4878317143963f6cbeadbe520b4ef4b195
-- 
2.21.0

